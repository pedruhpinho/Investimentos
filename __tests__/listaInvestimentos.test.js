import React from 'react';
import renderer from 'react-test-renderer';
import ListaInvestimentos from '../src/listaInvestimentos';

it('rederizou corretamente', () => {
    const tree = renderer.create(<ListaInvestimentos />).toJSON();
    expect(tree).toMatchSnapshot();
})