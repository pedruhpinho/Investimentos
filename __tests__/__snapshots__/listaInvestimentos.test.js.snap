// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rederizou corretamente 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#f4f4f4",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#f4f4f4",
        "flexDirection": "row",
        "padding": 20,
      }
    }
  >
    <Text>
      INVESTIMENTOS
    </Text>
    <View
      style={
        Object {
          "marginLeft": "auto",
        }
      }
    >
      <Text>
        R$
      </Text>
    </View>
  </View>
</View>
`;
